src/administration/integrations/gluescripts.md

```diff
@@ -0,0 +1,94 @@
+# GlueScript
+
+The GlueScript integration allows you to run JavaScript code to make your Platform.sh projects interact with external tools.
+
+## Setup
+
+### 1. Write the script
+
+Create a new `scripts` folder within the `.platform/` folder.
+
+```bash
+mkdir .platform/scripts
+```
+
+Create a new Javascript file inside the `scripts` folder with the following content:
+
+```javascript
+{
+  "type": "script",
+  "script": "<CODE>"
+}
+```
+
+where:
+* `CODE` is the Javascript code of the integration
+
+#### Custom Slack integration
+
+For example, the GlueScript can perform a custom Slack integration as follow:
+
+```javascript
+var message = activity.text;
+var color;
+
+if (activity.state == 'complete') {
+  if (activity.result == 'success') {
+    color = '#66c000';
+  } else {
+    color = '#ff0000';
+  }
+}
+
+var resp = fetch(
+  '<SLACK_PAYLOAD_URL>',
+  {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      'attachments': [{
+        'fallback': message,
+        'color': color,
+        'fields': [{
+          'value': message,
+        }],
+      }],
+    }),
+  }
+);
+
+if (!resp.ok) { throw 'Failed'; }
+```
+
+where:
+* `SLACK_PAYLOAD_URL` is the URL provided by Slack for your custom webhook
+
+### 2. Commit the script
+
+Commit the script to Git:
+
+```bash
+git add .platform/scripts/SCRIPT.js
```

Comment:
It's unclear: this file has this?

```javascript
{
    "type": "script",
    "script": "var message = activity.text;
var color;
// rest of the code..."
}
```

?

In that case, it's rather a JSON file than a JS file, no?

Reply:
JS is fine based on the syntax color scheme of my code editor. When the file is saved as JSON, it doesn't recognize the syntax :)

Reply:
Now that you're using the CLI, the file contents should be plain JS:

```
var message = activity.text;
var color;
// rest of the code...
```

Reply:
Wait, so is the file a Javascript embedded in JSON, or is it now just straight up Javascript?

I hope the latter, because the former is borderline abusive in terms of UX. :smile:

Reply:
JS and resolved in the "Other PR". Closing this one because all discussions were resolved there.